linebot/messaging_api/api_messaging_api.go

```diff
@@ -0,0 +1,3040 @@
+/**
+ * LINE Messaging API
+ * This document describes LINE Messaging API.
+ *
+ * The version of the OpenAPI document: 0.0.1
+ *
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+/**
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+//go:generate python3 ../../generate-code.py
+
+package messaging_api
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"net/url"
+	"path"
+	"strings"
+
+	"github.com/line/line-bot-sdk-go/v7/linebot"
+)
+
+// Deprecated: Use OpenAPI based classes instead.
+type MessagingApiAPI struct {
+	httpClient   *http.Client
+	endpoint     *url.URL
+	channelToken string
+	ctx          context.Context
+}
+
+// MessagingApiAPIOption type
+type MessagingApiAPIOption func(*MessagingApiAPI) error
+
+// New returns a new bot client instance.
+// Deprecated: Use OpenAPI based classes instead.
+func NewMessagingApiAPI(channelToken string, options ...MessagingApiAPIOption) (*MessagingApiAPI, error) {
```

Comment:
Is this a mistake...? The new one doesn't seem to be deprecated...? (I guess generating code again resolve this)

Reply:
I regenerated code and it was gone :)